src/Fixtures/Crawlers.php

```diff
@@ -1054,6 +1054,7 @@ class Crawlers extends AbstractProvider
         'TrapitAgent',
         'Trendiction',
         'Trendsmap Resolver',
+        'TrendsmapResolver',
```

Comment:
Could you please remove `Trendsmap Resolver` and `Trendsmap` and replace them both with `Trendsmap`. That way it's a single regex looking for both user agents.